pkg/endpoint/endpoint.go

```diff
@@ -284,17 +284,19 @@ type Endpoint struct {
 	Consumable *policy.Consumable `json:"-"`
 
 	// L4Policy is the L4Policy in effect for the
-	// endpoint. Normally it is the same as the Consumable's
+	// endpoint. Normally, it is the same as the Consumable's
```

Comment:
Unrelated to the PR, but "Normally" could be more specific, for instance "outside of policy recalculation".